src/backend/src/services/onboarding.services.ts

```diff
@@ -60,29 +65,36 @@ export default class OnboardingServices {
    * @param organization the organization of the checklists
    * @returns all the checklists for the given teamType Ids
    */
-  static async getTeamTypeChecklists(teamTypeIds: string[], organization: Organization) {
-    if (teamTypeIds.length === 0) {
-      throw new HttpException(400, 'teamTypeIds cannot be empty');
+  static async getUsersChecklists(userId: string, organization: Organization) {
+    const user = await prisma.user.findUnique({ where: { userId }, include: { teamsAsMember: true } });
+    if (!user) {
+      throw new NotFoundException('User', userId);
     }
 
-    const teamTypes = await prisma.team_Type.findMany({
-      where: { teamTypeId: { in: teamTypeIds } }
-    });
-
-    if (teamTypes.length !== teamTypeIds.length) {
-      throw new HttpException(400, 'One or more inavlid teamTypeIds');
-    }
+    const teamTypeIds = user.teamsAsMember.map((team) => team.teamTypeId).filter((id): id is string => id !== null);
```

Comment:
why do you need this weird id is string here

Reply:
if I dont have that it assumes id can still be null even though I am filtering it out so when I put it in the findMany it errors without it